daemon/api_test.go

```diff
@@ -2446,6 +2446,50 @@ func (s *apiSuite) TestPostSnapCohortRandoAction(c *check.C) {
 	}
 }
 
+func (s *apiSuite) TestPostSnapLeaveCohortRandoAction(c *check.C) {
```

Comment:
The testname contains "rando" ? What does that mean?

Reply:
noun
noun: rando; plural noun: randos

1.
    a person one does not know, especially one regarded as odd, suspicious, or engaging in socially inappropriate behaviour.
    "some rando asked to take her photo"

Reply:
I learned a new word!